DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices
+{
+    using System;
+    using System.Data;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.Register;
+
+    public interface IRegistrationDataService
+    {
+        string RegisterDelegate(DelegateRegistrationModel delegateRegistrationModel);
+    }
+
+    public class RegistrationDataService: IRegistrationDataService
+    {
+        private readonly IDbConnection connection;
+
+        public RegistrationDataService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public string RegisterDelegate(DelegateRegistrationModel delegateRegistrationModel)
+        {
+            var values = new
+            {
+                FirstName = delegateRegistrationModel.FirstName,
+                LastName = delegateRegistrationModel.LastName,
+                Email = delegateRegistrationModel.Email,
+                CentreID = delegateRegistrationModel.Centre,
+                JobGroupID = delegateRegistrationModel.JobGroup,
+                Active = 1,
+                Approved = 0,
+                Answer1 = "",
+                Answer2 = "",
+                Answer3 = "",
+                Answer4 = "",
+                Answer5 = "",
+                Answer6 = "",
+                AliasID = "",
+                ExternalReg = 0,
+                SelfReg = 0,
+                NotifyDate = DateTime.Now,
+                Bulk = 1

```

Comment:
Is this correct? I'd expect the "SelfReg" to be true when someone goes through the form. Also, do you know what the "Bulk" is (I can double check with Kevin if you're not sure)?

Reply:
Yeah I actually meant to ask about those! SelfReg should be true, that makes sense, and yes I don't know what bulk is for.

Reply:
I suspect "Bulk" is when the admins register delegates in bulk, but I'm not 100% sure. I will chat to Kevin about this tomorrow, but for now, could we keep SelfReg as 1 and Bulk as 0?